website/src/main/java/com/google/step/data/CloudSQLManager.java

```diff
@@ -168,6 +177,31 @@ public void executeStatement(String query) throws SQLException {
     stmt.executeUpdate(query);
   }
 
+  public ResultSet getFirstUploadOrg(String tableName) throws SQLException {
+    String query = String.format("SELECT * FROM %s ORDER BY id LIMIT 0, 1", tableName);
```

Comment:
if you are just getting the first element, you don't need to specify the offset as 0, so "LIMIT 1" is sufficient